Resources/Prototypes/Species/slime.yml

```
@@ -36,19 +36,19 @@
       points: 1
       required: false
     Chest:
-      points: 1
+      points: 4 # imp
```

```suggestion
      points: 4 # imp 1<4
```
Note original value for future maintenance work.